### PR TITLE
chore(compass-components): update ai entry text and icon, align styles for multi-line COMPASS-7207

### PR DESCRIPTION
**Pull request by Anemy**
COMPASS-7207 COMPASS-7222

This PR updates the ai entry. Instead of `Ask AI` with the robot icon, it's now `Generate query` and `Generate aggregation` with the chat-like icon.
Also fixed the multi-line size. So it now aligns nicely with the top:
Screenshot 2023-09-18 at 9 30 29 AM

| light | dark |
| - | - |
| Screenshot 2023-09-18 at 9 26 40 AM | Screenshot 2023-09-18 at 9 26 57 AM |
| Screenshot 2023-09-18 at 9 26 44 AM | Screenshot 2023-09-18 at 9 26 54 AM |
| Screenshot 2023-09-18 at 9 27 09 AM | Screenshot 2023-09-18 at 9 27 03 AM |

### CODE REVIEW

**Comment by Anemy** [comment]
Just heard the icon might change again, so those parts are subject to change.

File: packages/compass-components/src/components/generative-ai/ai-entry-svg.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    />
+    <path
+      d="M12.5469 11.2302L12.3159 11.9234C12.2158 12.2237 11.9801 12.4594 11.6797 12.5595L10.9866 12.7906C10.7573 12.867 10.7573 13.1913 10.9866 13.2677L11.6797 13.4988C11.9801 13.5989 12.2158 13.8346 12.3159 14.1349L12.5469 14.8281C12.6233 15.0573 12.9476 15.0573 13.024 14.8281L13.2551 14.1349C13.3552 13.8346 13.5909 13.5989 13.8912 13.4988L14.5844 13.2677C14.8136 13.1913 14.8136 12.867 14.5844 12.7906L13.8912 12.5595C13.5909 12.4594 13.3552 12.2237 13.2551 11.9234L13.024 11.2302C12.9476 11.001 12.6233 11.001 12.5469 11.2302Z"
+      fill="#00684A"
+    />
+  </svg>
+);
+
+// Note: This is duplicated above for react.
+const getAIEntrySVGString = () => `<svg
```
Comment (gribnoysup) [inline packages/compass-components/src/components/generative-ai/ai-entry-svg.tsx:72]: Does this need dark mode support too?
```suggestion
const getAIEntrySVGString = (darkMode = false) => `<svg
```

**Review by lerouxb — APPROVED** [review]

**Review by gribnoysup — APPROVED** [review]

File: packages/compass-components/src/components/generative-ai/ai-entry-svg.tsx (hunk 1 above)
Comment (Anemy) [reply packages/compass-components/src/components/generative-ai/ai-entry-svg.tsx:72]: It's covered in the container for it. A bit hacky, but does the job.

Comment (gribnoysup) [reply packages/compass-components/src/components/generative-ai/ai-entry-svg.tsx:72]: Do we need to follow up to remove the hack?

**Pull request merged**